modules/openapi-generator/src/main/resources/aspnetcore/2.1/Startup.mustache

```diff
@@ -109,7 +109,13 @@ namespace {{packageName}}
 
                     //TODO: Or alternatively use the original Swagger contract that's included in the static files
                     // c.SwaggerEndpoint("/openapi-original.json", "{{#appName}}{{{appName}}}{{/appName}}{{^appName}}{{packageName}}{{/appName}} Original");
-                }){{/useSwashbuckle}};
+                }){{/useSwashbuckle}};{{^useDefaultRoutng}}
+            	app.UseHttpsRedirection();
```

Comment:
This is already done at the beginning of the method.

Reply:
Removed the duplicate